options/options.cc

```diff
@@ -491,7 +491,6 @@ DBOptions* DBOptions::OldDefaults(int rocksdb_major_version,
     delayed_write_rate = 16 * 1024U * 1024U;
```

Comment:
I missed this in the code review.
I probably still should set max_background_compactions and max_background_flushes to the original value here. Also mention it in DEFAULT_OPTIONS_HISTORY.md.